packages/portalnetwork/test/examples/portalClient.spec.ts

```diff
@@ -0,0 +1,164 @@
+import { PortalNetwork } from '../../../portalnetwork/src'
```

Comment:
I'm not 100% clear what this is but I think my original request wasn't clear.  We don't need a new test with lots of mocks.  What I was hoping you would do is move the `packages/cli/scripts/portalClient.ts` script to `packages/portalnetwork/examples/portalClient.ts`.  The original script you wrote is actually pretty cool as is for a proof of concept.  

This test script as written isn't really ideal.  I don't think we need a new test specificaly for the default settings and would be okay merging them in without additional tests.

Reply:
Updated